src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -253,6 +292,23 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
     }
   }
 
+  const prepareTtl = (ttl?: number) => {
+    if (!ttl && !isFeatureSupportedForVersion(LedgerCryptoProviderFeature.ALLEGRA)) {
+      throw Error(Errors.LedgerOptionalTTLNotSupported)
+    }
+    return ttl && ttl.toString()
+  }
+
+  const prepareValidityIntervalStart = (validityIntervalStart?: number) => {
```

Comment:
validityIntervalStart should technically be a BigInt, not number, Ledger/CDDL specifies it as int64

Reply:
My bad, I assumed ttl and validity interval start are numbers, thnx :+1:

Reply:
Fixed